wsd/DocumentBroker.hpp

```diff
@@ -484,8 +485,13 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
             _status = newStatus;
         }
 
+        /// Flag close-requested. Cannot be reset.
+        void setCloseRequested() { _closeRequested = true; }
+        bool isCloseRequested() const { return _closeRequested; }
+
     private:
         Status _status;
+        bool _closeRequested; //< Owner-Termination flag.
```

Comment:
Not std::atomic any more (?)

Reply:
This is set from ClientSession, which is invoked from the poll. There are no other threads to change this value (it's also set only, never reset). So my expectation is that there is no more concern with threading or visibility across threads.

Will triple check to make sure.

Reply:
I restored atomic because this is invoked from Admin and also to make this as identical to the original code as possible.